### PR TITLE
Php

### PR DESCRIPTION
**Pull request by tpeyrard**
Add code for PHP client generation.

The client won't be built and pushed, because I didn't add the language PHP in both scripts.
I'll do it in another commit.

### CODE REVIEW

File: build-php/build.gradle [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    dependencies {
+        classpath "org.openapitools:openapi-generator-gradle-plugin:4.1.3"
+    }
+}
+
+apply plugin: 'java-library'
+apply plugin: 'org.openapi.generator'
+
+ext {
+    artifactId = "python-client-generator"
```
Comment (rmadisonhaynie) [inline build-php/build.gradle]: This needs to be updated right?

File: build-php/build.gradle [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    groupId = "com.criteo.mapi"
+    version = '1.0.0'
+    description = 'Criteo Generator for Python Client'
```
Comment (rmadisonhaynie) [inline build-php/build.gradle]: Here

File: build-php/resources/templates/composer.mustache [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    "homepage": "https://api.criteo.com",
+    "license": "Apache-2.0",
+    "authors": [
+        {
+            "name": "Criteo",
+            "homepage": "https://github.com/criteo/criteo-php-marketing-sdk"
+        }
+    ],
+    "require": {
+        "php": ">=7.1",
```
Comment (rmadisonhaynie) [inline build-php/resources/templates/composer.mustache]: here greater or equal 7.1

File: build-php/resources/templates/README.mustache [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+- Build date: {{generatedDate}}
+{{/hideGenerationTimestamp}}
+- Build package: {{generatorClass}}
+{{#infoUrl}}
+For more information, please visit [{{{infoUrl}}}]({{{infoUrl}}})
+{{/infoUrl}}
+
+## Requirements
+
+PHP 5.5 and later
```
Comment (rmadisonhaynie) [inline build-php/resources/templates/README.mustache]: Here greater or equal 5.5

File: build-php/build.gradle (hunk 1 above)
Comment (tpeyrard) [reply build-php/build.gradle]: Updated :)

File: build-php/resources/templates/README.mustache (hunk 4 above)
Comment (tpeyrard) [reply build-php/resources/templates/README.mustache]: Nice catch. That's from the official template. I'll update the README to match the composer file, i.e 7.1.

**Pull request merged**